pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -77,7 +66,7 @@ func Destroy(dir string) (err error) {
 			return errors.Wrap(err, "Terraform apply")
 		}
 	case libvirt.Name:
-		_, err = terraform.Apply(tempDir, platform, extraArgs...)
+		_, err = terraform.Apply(tempDir, platform, append(extraArgs, "-var=bootstrap_dns=false")...)
```

Comment:
A comment above this line would be nice, like:

```
// First remove the bootstrap node from DNS
```

Reply:
Done.